blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/PreaggregateConversions.java

```diff
@@ -43,6 +46,10 @@
     
     public static Collection<IMetric> buildMetricsCollection(AggregatedPayload payload) {
         Collection<IMetric> metrics = new ArrayList<IMetric>();
+        long nowMillis = new DateTime().getMillis();
+        if (nowMillis - payload.getTimestamp() > millisIn10Minutes) {
```

Comment:
if the rollup is 10 minutes or older we are considering it "delayed"?

We might want to make this bench mark configurable instead of hard coding it, in case we want to tweek the value.